docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -204,7 +205,7 @@ Preparing For a Release
 
    .. prompt:: bash [vagrant@localhost]$
 
-      ~/flocker-${VERSION}/admin/publish-docs --doc-version ${VERSION}
+      admin/publish-docs --doc-version ${VERSION}
```

Comment:
:question: ...this is now the version of `publish-docs` from the release branch....which may be very out-of-date if we're working a patch release of an old version.
Not an issue right now though.